compiler/optimizer/SinkStores.cpp

```diff
@@ -3415,7 +3415,10 @@ bool TR_TrivialSinkStores::sinkStorePlacement(TR_MovableStore *store,
          _symbolsKilledInBlock[currentBlockNumber]->print(comp());
          traceMsg(comp(),"\n");
          traceMsg(comp(),"         blockNumber %d symbols used:  ",currentBlockNumber);
-         _symbolsUsedInBlock[currentBlockNumber]->print(comp());
+         if (_symbolsUsedInBlock[currentBlockNumber])
```

Comment:
Failure to print the diagnostic because of this if should still print something so this needs an else that prints in some way consistent with the trace formatting around it.